assets/packages/questions/ranking/scripts/ranking.js

```diff
@@ -19,7 +19,7 @@ var RankingQuestion = function (options) {
         rankingID = "javatbd" + rankingName;
 
     //define HTML snippets
-    var screenReader = "<div class='sr-only'>" + $('#question' + questionId + ' .em_default').html() + "</div><div aria-hidden='true'>" + LSvar.lang.rankhelp + "</div>"
+    var screenReader = "<div class='sr-only'>" + $('#question' + questionId + ' .em_default').html() + "</div><div aria-hidden='true'><span class='fa fa-exclamation-circle' aria-hidden='true'></span> " + LSvar.lang.rankhelp + "</div>"
```

Comment:
1. It seems to be the screenreader help, not the help for other user.
2. Maybe best is to copy current help and replace the text : then it came from twig file (template can replace this twig file)

Reply:
> 1. It seems to be the screenreader help, not the help for other user.

Well. Not sure about that. The icon appears after this.
![image](https://github.com/LimeSurvey/LimeSurvey/assets/4942495/99f783a5-231c-494e-96d6-d43a481ecb31)

> Maybe best is to copy current help and replace the text : then it came from twig file (template can replace this twig file)

The ticket is just about showing the icon. This is v5. Not sure it is worth here.